src/uu/split/src/split.rs

```diff
@@ -138,6 +147,12 @@ size is 1000, and default PREFIX is 'x'. With no INPUT, or when INPUT is
     settings.input = input;
     settings.prefix = prefix;
 
+    settings.filter = if let Some(filter) = matches.opt_str("filter") {
```

Comment:
are you sure there isn't a better way to do that?
like with unwrap_or_else ?

Reply:
replaced